### ISSUE TITLE
TextInput editable not works

### ISSUE THREAD

**Issue by sm2017** [callstack/react-native-paper#865]
### Current behaviour

TextInput editable not works

### Expected behaviour
TextInput editable and disabled is different behavior 

### Code sample

```js
import * as React from 'react';
import { TextInput } from 'react-native-paper';

export default class MyComponent extends React.Component {
  state = {
    text: ''
  };

  render(){
    return (
      <TextInput
        label='Email'
        value={this.state.text}
        editable={false}
        onChangeText={text => this.setState({ text })}
      />
    );
  }
}
```

### What have you tried

see https://github.com/callstack/react-native-paper/blob/master/src/components/TextInput.js#L647 , you can see here not disabled is equal to editable

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        |
| react-native          |
| react-native-paper    |
| node                  |
| npm or yarn           |
| expo sdk              |

**Comment by sm2017** [comment]
I send https://github.com/callstack/react-native-paper/pull/866

**Comment by raajnadar** [comment]
A discussion is already going on https://github.com/callstack/react-native-paper/issues/737

**Comment by jbinda** [comment]
fixed in #1069
